### PR TITLE
Add Windows/WSL instructions

### LINKED ISSUE #232: Add docs to install WSL on Windows
Opening this issue as something we will _very likely_ include in the technical set up docs. It's not currently reflected in #61, but once we can actually test out the windows machine and final.final confirm this is the right choice, we can add to IA and write this docs page.

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #232 

This PR initiates Windows WSL setup instructions:

- I added a section to the setup index page
- I added a WSL2 installation page
- I added a section to the terminal page for WSL2

Although this is ready for review, it is being filed as a Draft since we do not want to merge it yet. Coming up next will be a series of stacked PRs on top of this one. If/when we are ready to include WSL instructions, we can then merge this PR (which is expected to contain all the stacked content).

### CODE REVIEW

**Comment by sjspielman** [comment]
Noting the links check failure here is expected!

File: docs/technical-setup/install-wsl2.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ```sh
+    wsl --install
+    ```
+
+1. WSL2 will now install.
+Along the way, you may get prompts asking if you allow the app to make changes to your device.
+Always click "Yes" when you see these prompts.
+
+1. Once WSL2 has finished installing, open the new Ubuntu app if it does not open automatically.
+    - Ubuntu should prompt you to create a username and password which represent your WSL2 credentials; these are _different_ from the username and password you already have set up on the Windows side of your computer.
```
Comment (jashapiro) [inline docs/technical-setup/install-wsl2.md:34]: I expect many people will use the same username and password, and really that is fine. But I think we can be more clear about what different means here. Or rephrase and independent?
```suggestion
    - Ubuntu should prompt you to create a username and password which represent your WSL2 credentials. 
      These are _independent_ of the username and password you already have set up on the Windows side of your computer.
      Changing one will not affect the other, but you can use the same username for both if you would like.
```

**Comment by sjspielman** [comment]
Attempted a rephrase..what do you think? Also note that I ran spell check locally and made a couple changes accordingly.

**Comment by sjspielman** [comment]
> Also note that I ran spell check locally and made a couple changes accordingly.

Hm, should we turn back on the spellcheck action on PRs for changes in `docs/` only?

File: docs/technical-setup/index.md [hunk 2]
```diff
@@ -7,10 +7,22 @@ This section provides instructions for technical aspects of set up, including:
...
-!!! warning
-    At this time, the OpenScPCA project only supports macOS and Linux operating systems.
-    Instructions are therefore geared towards using these platforms.
+## Special considerations for Windows users
 
-    If you would like to see Windows supported, please [let us know by filing an issue](https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/new?assignees=&labels=docs-request&projects=&template=04-docs-request.yml&title=Docs+request%3A)!
+Due to the OpenScPCA project's technical environment needs, Windows users will [need to install and use Windows Subsystem for Linux 2 (WSL2)](./install-wsl2.md).
+
+
+[WSL2](https://learn.microsoft.com/en-us/windows/wsl/about) lets you run a separate Linux (Ubuntu) environment on your Windows computer and provides a separate Linux file system that is separate from your regular Windows file system.
```
Comment (jashapiro) [inline docs/technical-setup/index.md:15]: I tried separating the two parts here, and not saying "separate" separately in every clause. We should probably link to something about the two file systems...
```suggestion
[WSL2](https://learn.microsoft.com/en-us/windows/wsl/about) lets you run a separate Linux (Ubuntu) environment on your Windows computer.
This includes a separate Linux file system that is mostly distinct from your regular Windows file system.
```

File: docs/technical-setup/install-wsl2.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+If you do not have at least these Windows versions, you must use [Lightsail for Research](../software-platforms/aws/index.md#lightsail-for-research-virtual-computing-with-aws) instead.
+
+
+## Install WSL2
+
+These instructions will provide you with both WSL2 and the Ubuntu app:
+
+- WSL2 allows your computer to have a separate Linux subsystem
+- Ubuntu is the default Linux distribution that gets installed when you install WSL2
+    - The Ubuntu app also acts as the Linux [the terminal interface](../software-platforms/general-tools/using-the-terminal.md).
```
Comment (jashapiro) [inline docs/technical-setup/install-wsl2.md:20]:
```suggestion
    - The Ubuntu app also acts as the Linux [terminal interface](../software-platforms/general-tools/using-the-terminal.md).
```

**Comment by sjspielman** [comment]
Not 100% sure how I feel about the link I added, what do you think?

File: docs/technical-setup/install-wsl2.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Installation instructions
+
+These instructions will provide you with both WSL2 and the Ubuntu app:
+
+- WSL2 allows your computer to have a separate Linux subsystem
+- Ubuntu is the default Linux distribution that gets installed when you install WSL2
+    - The Ubuntu terminal application also acts as the Linux [terminal interface](../software-platforms/general-tools/using-the-terminal.md).
+    - The Ubuntu terminal is the main way you will interact with the Linux side of your computer.
+        - To learn more about the relationship between your Linux and Windows file systems, please refer to this article on [Working across Windows and Linux file systems](https://learn.microsoft.com/en-us/windows/wsl/filesystems).
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply docs/technical-setup/install-wsl2.md:22]: > I think this link is okay, but as you say, not great. Quick searching didn't find much better, so leave it.

Yup, pretty much exactly my experience...

**Comment by sjspielman** [comment]
The spelling failure should be fixed once #381 (and possibly #385) goes in.

**Pull request merged**